spaceship.zsh

```diff
@@ -738,6 +746,25 @@ spaceship_julia() {
     "$SPACESHIP_JULIA_SUFFIX"
 }
 
+# JAVA
+# Show current Java version
+spaceship_java() {
```

Comment:
Hmm. [jenv](https://github.com/gcuisinier/jenv) seems quite stable and popular. Consider adding this too. You can show `jenv` version if `jenv` is installed. Otherwise you can show `java --version`, of course only for Java-related projects.

Check out how other sections of Spaceship works for inspiration.